src/controller/python/chip/clusters/Attribute.py

```diff
@@ -795,14 +803,23 @@ def _handleDone(self):
 
     def handleDone(self):
         self._event_loop.call_soon_threadsafe(self._handleDone)
-
+        
     def handleReportBegin(self):
-        pass
-
+        self._handleReportBegin()
+        
     def handleReportEnd(self):
-        # self._event_loop.call_soon_threadsafe(self._handleReportEnd)
         self._handleReportEnd()
 
+    def _handleNotifySubscriptionStillActive(self):
+        if self._notify_subscription_still_active_callback:
+            self._notify_subscription_still_active_callback()
+
+    def handleNotifySubscriptionStillActive(self):
```

Comment:
The internal function is set with
```
    def register_notify_subscription_still_active_callback(self, callback):
        self._notify_subscription_still_active_callback = callback  
```
So that code block first checks if the callback was set, and executes if so

Reply:
These two can be simplified though
```
    def _handleNotifySubscriptionStillActive(self):
        if self._notify_subscription_still_active_callback:
            self._notify_subscription_still_active_callback()

    def handleNotifySubscriptionStillActive(self):
        self._handleNotifySubscriptionStillActive()
```
Will do that.